bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import subprocess
+from pathlib import Path
+
+def uniffi_gen(executable: str, *args):
+    args = list(str(a) for a in args)
+    uniffi = [executable]
+    uniffi.extend(args)
+    print(f'Running {" ".join(uniffi)}')
+    subprocess.run(uniffi, check=True)
+
+def uniffi_gen_go(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-go", udl, "-o", output_dir)
+
+
+def uniffi_gen_cs(udl: Path, output_dir: Path, config_file: Path):
+    uniffi_gen("uniffi-bindgen-cs", udl, "-o", output_dir, "-c", config_file)
+
+
+def uniffi_gen_swift(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "swift", "-o", output_dir)
```

Comment:
So this requires the `uniffi-bindgen` to be installed on the machine? I was following the guide https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html and it says that you need to create this `uniffi-bindgen` package in the repo. Am I missing something? Maybe @Lipt0nas knows more about how it works

Reply:
Good :ok: